swagger.json

```diff
@@ -2294,6 +2402,65 @@
                 }
             }
         },
+        "FleetVehiclesLocations": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "integer",
+                "description": "ID of the vehicle.",
+                "format": "int64",
+                "example": 112
+              },
+              "name": {
+                "type": "string",
+                "description": "Name of the vehicle.",
+                "example": "Truck A7"
+              },
+              "locations": {
+                "$ref": "#/definitions/FleetVehicleLocations"
+              }
+            }
+          }
+        },
+        "FleetVehicleLocations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FleetVehicleLocation"
+          }
+        },
+        "FleetVehicleLocation": {
+          "type": "object",
+          "description": "Contains the location and speed of a vehicle at a particular time",
+          "properties": {
+            "location": {
+              "type": "string",
+              "description": "The best effort (street,city,state) for the latitude and longitude.",
+              "example": "525 York, San Francisco, CA"
+            },
+            "latitude": {
+              "type": "number",
+              "description": "The latitude of the location in degrees.",
+              "example": "37"
+            },
+            "longitude": {
+              "type": "number",
+              "description": "The longitude of the location in degrees.",
+              "example": "-122"
+            },
+            "speedMilesPerHour": {
+              "type": "number",
+              "description": "The speed calculated from GPS that the asset was traveling at in miles per hour.",
+              "example": "35"
```

Comment:
We should update this to be 35.3 to illustrate that it's floating point and not integer.

Reply:
sg

Reply:
done